maufbapi/mqtt/conn.py

```diff
@@ -138,7 +138,7 @@ def _form_client_id(self) -> bytes:
         #                     RealtimeTopic.MARK_THREAD_READ_RESPONSE,
         #                     ]
         subscribe_topics = [RealtimeTopic.MESSAGE_SYNC, RealtimeTopic.REGION_HINT,
-                            RealtimeTopic.SEND_MESSAGE_RESP,
+                            RealtimeTopic.SEND_MESSAGE_RESP, RealtimeTopic.ORCA_PRESENCE,
```

Comment:
This subscribe should probably be conditional, especially since it's not in the default topics that messenger on android subscribes to

Reply:
The subscription is now dependent on the 'bridge.presence' flag in the config.